estimate-web/src/actions/estimation.js

```diff
@@ -75,6 +83,14 @@ export const receiveWorkItemScored = args => ({
     iterationPath: args.groupName
 });
 
+export const REQUEST_WORKITEM_REFRESH_COMMENTS = "ESTIMATION/REQUEST_WORKITEM_REFRESH_COMMENTS";
```

Comment:
Is there a difference between refresh / get comments? Seems like it's a very similar action :-)

Reply:
`GET_COMMENTS` will be executed once on each client when a workItem is loaded.
`REFRESH_COMMENTS` on the other hand is an action that's distributed to the other clients via SignalR (see estimation saga). After I've added a comment on my client, the app will tell the other clients to refresh, so the comment is immediately visible to everyone.

Reply:
Sounds good!